engine/src/main/java/com/datatorrent/stram/util/LoggerUtil.java

```diff
@@ -278,4 +280,37 @@ public static synchronized void changeLoggersLevel(@Nonnull Map<String, String>
     }
     return ImmutableMap.copyOf(matchedClasses);
   }
+
+  /**
+   * Returns logger log file name
+   * @return logFileName
+   */
+  public static String getLogFileName()
+  {
+    Enumeration e = LogManager.getRootLogger().getAllAppenders();
+    while (e.hasMoreElements()) {
+      Appender appender = (Appender)e.nextElement();
+      if (appender instanceof FileAppender) {
+        return ((FileAppender)appender).getFile();
```

Comment:
Apex supports custom log4j.properties that may have multiple FileAppenders and may not use ${hadoop.root.logger} as a root logger. It will be good to support custom log4j.properties or at minimum have an ability to add such support in the future without changing API.